### PR TITLE
[v-play] enable translations

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/Silica4v-play/resources-Silica4v-play.qrc [hunk 1]
```diff
@@ -14,6 +14,7 @@
         <file>qmldir</file>
         <file>SearchField.qml</file>
         <file>SilicaListView.qml</file>
+        <file>TruncationMode.qml</file>
```
Comment (micuintus) [inline qml/Silica4v-play/resources-Silica4v-play.qrc]: Oh man, that's a relief. The error messages were killing me!

File: resources.qrc [hunk 2]
```diff
@@ -15,7 +15,9 @@
         <file>qml/pages/VenueList.qml</file>
         <file>qml/pages/VenueMapPage.qml</file>
         <file>qml/harbour-berlin-vegan.qml</file>
+        <file>translations/harbour-berlin-vegan-de.qm</file>
```
Comment (micuintus) [inline resources.qrc]: Nice! I have actually been confused all the time how and why the translations were already working on Sailfish.
I assume they have some other mechanism in place that deploys the QM files to the RPM/device.
Are these qm files used for QML, at all?

File: qml/pages/VenueList.qml [hunk 3]
```diff
@@ -88,8 +88,8 @@ Page {
 
         property Component heading:
         PageHeader {
-                         //% "Vegan food nearby"
-            title: qsTrId("id-vegan-food-nearby")
+                         //% "Berlin Vegan Guide"
```
Comment (micuintus) [inline qml/pages/VenueList.qml]: So you don't like the "Vegan food nearby"? ;)

**Comment by micuintus** [comment]
Very nice work! Thanks!

Unfortunately, now only the id "id-berlin-vegan-guide" is shown when you disable the search on Sailfish --- both with English as well as German being set as the language. :/ I think we always need to supply a translation on Sailfish. That ```source``` thing only does not seem to be working there. :|

Furthermore, including the new one, we now also have three ids that basically reflect the app name: 
```
    <message id="id-berlin-vegan-guide">
    <message id="id-berlin-vegan">
    <message id="id-cover-berlin-vegan">
```
I'd suggest rather reducing the number of these kind of ids instead of increasing them ;).

On a more general note, I wonder whether "Berlin Vegan Guide" is the term we would like to run with in the long-term. Actually I have already spend quite some time thinking about the app name. ;) The Berlin-Vegan group consistently uses "Berlin-Vegan" (with the hyphen and capital B and capital V) as their name --- on the website, for the Android app, for the name of the Facebook page. To be honest, I personally don't like this name too much. In particular, I find the hyphen a bit weird. But since this name is used so consistently, I'd like to suggest sticking with "Berlin-Vegan" for now.

File: resources.qrc (hunk 2 above)
Comment (jmastr) [inline resources.qrc]: Yes, there are necessary. Without them one only gets the ID strings.

**Pull request merged**